docs/general/faq.md

```diff
@@ -212,12 +212,11 @@ ten billion) Plancks, meaning that the new DOT was valued at ten decimal places.
 
 ### What is the inflation rate of the DOT?
 
-The inflation rate is approximately 10% per year.
+The inflation rate is approximately 120,000,000 DOT per year.
 
-A portion of the inflation is rewarded to validators for performing their duties, while another
-portion may go directly to the treasury. The exact percentage that goes into both varies and is
-based on the amount of DOT that are staked. Please see the article on
-[inflation](../learn/learn-staking.md/#inflation) for more information.
+The 85% of inflation is rewarded to validators for performing their duties, while the 15% goes to
```

Comment:
Note that this 15% is now hardcoded, but used to be something that we intended to parameterize with `MaxStakingReward` in Polkadot.

Reply:
yeah, this is why I did not mention it for now. I know that the current inflation fix it temporary and at some point parameters will be exposed to gov. We can add this info once is done.

Reply:
> Note that this 15% is now hardcoded, but used to be something that we intended to parameterize with `MaxStakingReward` in Polkadot.

Yes, the goal is to have this adjustable. Since the fixed DOT issuance was not something that we could do with the current code (except hardcoding it), we had to take a shortcut. I'll make an issue to reintroduce adjustability via governance maybe with integration of the parameters pallet.